src/storage/src/render/sources.rs

```diff
@@ -236,6 +241,7 @@ fn render_source_stream<G>(
     Collection<G, Row, Diff>,
     Collection<G, DataflowError, Diff>,
     Vec<Rc<dyn Any>>,
+    Option<Stream<G, (WorkerId, OutputIndex, HealthStatusUpdate)>>,
```

Comment:
Instead of an `Option<Stream>`, just have the empty cases use `Stream::empty` (its not in the version of timely that is on `docs.rs`)

Reply:
Ah, that would be better. Let me update.

Reply:
done!